README.md

```diff
@@ -41,8 +41,17 @@ To try out (you will need Docker installed):
 ```bash
 git clone https://github.com/brokenhandsio/SteamPressExample.git
 cd SteamPressExample
-./setupDB.sh
+./bin/setupDB.sh
 swift run
 ```
 
 This will create a site at http://localhost:8080. The blog can be found at http://localhost:8080/blog/ and you can login at http://localhost:8080/blog/admin/. The first time you visit the login a user will be created and the details printed to the console.
+
+**Note**: If you receive an error such as "Thread 1: Fatal error: Error raised at top level...", then it is likely you are already running an instance of PostgreSQL on your machine. To try and resolve:
```

Comment:
This could be due to a number of errors, such as something already running on port 8080 so we probably need to quality this better